src/openrct2-ui/windows/Scenery.cpp

```diff
@@ -720,6 +720,15 @@ class SceneryWindow final : public Window
             const auto lastTabWidget = &widgets[WIDX_SCENERY_TAB_1 + lastTabIndex];
             windowWidth = std::max<int32_t>(windowWidth, lastTabWidget->right + 3);
 
+            if (_tabEntries[_tabEntries.size() - 2].IsMisc())
```

Comment:
It might be more robust if we make a change like this:
```suggestion
            if (GetSceneryTabInfoForMisc() != nullptr)
```

Unfortunately there isn't anything like this to get the index. You might also be able to exclude the `!= nullptr`, but I like to be explicit.